src/mol-script/transpilers/rasmol/special_keywords.ts

```diff
@@ -0,0 +1,232 @@
+/**
```

Comment:
I removed `special_keywords.ts` and `special_operators.ts` because they are not needed.  And I found these filenames were not good, I changed `special_properties` to `macroproperties ` as it's used to define selection macros. 

I can re-change the name if it's better to place hyphen between the words (`macro-properties.ts` rather than macroproperties.ts).